Source/Reporters/JunitXMLReporter.jsl

```diff
@@ -33,7 +33,148 @@ Example of JunitXML format:
 
 Define Class(
   "UtJunitXMLReporter",
-  Base Class( UtCollectingReporter ),
+  Base Class( "UtReporter" ),
+
+  // Variable: hash_results
+  // associative array of the test results
+  // test suite => test case => successes, failures, throws
+  hash_results = [=>Associative Array()];
```

Comment:
Assuming you want the default value to be an associative array, this should be `[=>[=>]]` instead. Due to the way that associative arrays are parsed, `[=>Associative Array()]` is an associative array with a default value of the expression `Associative Array()`.
```
aa = [=>Associative Array()];
Type(aa["foo"]);
/*:

"Expression"
```